docs/spec_draft.md

```diff
@@ -941,6 +942,68 @@ produces an implementation-defined value.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.dynamic_update_slice
+
+### Semantics
+
+Overwrites the `operand` tensor at `start_indices` with values from `update`
+tensor and produces a `result` tensor.
+
+More formally, `result[j0, ..., jR-1] = update[i0, ..., iR-1]` if
```

Comment:
Should we consider the following instead:


More formally, `result[j0, ..., jR-1]` is defined as
 * `jd = effective_start_indices[d] + id` where `effective_start_indices[d] = `
`clamp(start_indices[d], 0, dim(operand, d) - dim(update, d))`, 
* `operand[i0, ..., iR-1]` otherwise.

Reply:
I think you had a typo, I'll use this to borrow from your proposal:

More formally, `result[j0, ..., jR-1]` is defined as:
  * `update[i0, ..., iR-1]` if `jd = effective_start_indices[d] + id` where
    `effective_start_indices[d] = clamp(start_indices[d], 0, dim(operand, d) - `
    `dim(update, d))`
  * `result[i0, ..., iR-1] = operand[i0, ..., iR-1]` otherwise.

Reply:
Ah! my bad. Yes, this is what I meant.